resource-managers/kubernetes/integration-tests/dev/spark-rbac.yaml

```diff
@@ -49,4 +53,4 @@ subjects:
 roleRef:
   kind: ClusterRole
   name: spark-role
-  apiGroup: rbac.authorization.k8s.io
\ No newline at end of file
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
This is unrelated but I think it's reasonable reformat.

Reply:
I agree with you. The change is okay. Just a note: This is what your editor does although you aren't aware of it manually. If you do this in the opposite direction, it would get a negative feedback.
> I didn't change anything in here manually.